addons/prometheus/template/generate.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+set -euo pipefail
+
+VERSION=""
+function get_latest_release_version() {
+    VERSION=$(curl -I https://github.com/prometheus-operator/prometheus-operator/releases/latest | \
```

Comment:
does this version always track the helm version?

Reply:
hmm
yeah, there'll be a gap before it gets updated
let me see if there's a better solution there

Reply:
found a better solution